dags/daily_motivation_sender_dag.py

```
@@ -0,0 +1,186 @@
+from os import getenv
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def get_random_image(**kwargs):
+    """
+    Fetches a random image from Unsplash.
+
+    Returns:
+        dict: A dictionary containing the URL of the image and its description.
+              Returns None if there's an error or unexpected response.
+    """
+    UNSPLASH_ACCESS_KEY = getenv("UNSPLASH_ACCESS_KEY")
+    url = f"https://api.unsplash.com/photos/random?client_id={UNSPLASH_ACCESS_KEY}"
+    response = get(url)
+
+    if response.status_code != 200:
+        error(f"Error fetching image: {response.status_code}")
+        error(response.text)
+        return None
+
+    try:
+        data = response.json()
+
+        image_url = data.get("urls", {}).get("regular")
+        if not image_url:
+            error("Image URL not found in response:")
+            error(dumps(data, indent=4))
+            return None
+
+        image_description = data.get("alt_description")
+        if not image_description:
+            error("Image description not found in response:")
+            error(dumps(data, indent=4))
+            return None
+
+        info("Image fetched successfully")
+        return {"image_url": image_url, "image_description": image_description}
+
+    except JSONDecodeError as e:
+        error("Failed to parse JSON response:")
+        error(e)
+        return None
+
+
+def get_random_quote(**kwargs):
+    """
+    Fetches a random quote from Quotable.
+
+    Returns:
+        dict: A dictionary containing the quote content and author.
+              Returns None if there's an error or unexpected response.
+    """
+    QUOTABLE_API_URL = 'https://api.quotable.io/random'
+    response = get(QUOTABLE_API_URL)
+    if response.status_code != 200:
+        error(f"Error fetching quote: {response.status_code}")
+        error(response.text)
+        return None
```

It's better to raise an error in case of problems - this way you would see at which step some troubles have occurred